storage/badger_storage.go

```diff
@@ -171,15 +171,15 @@ func PerformanceBadgerOptions(dir string) badger.Options {
 	// This option will have a significant effect the memory. If the level is kept
 	// in-memory, read are faster but the tables will be kept in memory. By default,
 	// this is set to false.
-	opts.KeepL0InMemory = false
+	opts.KeepL0InMemory = true
```

Comment:
In practice, we saw this decrease read/write latency. In the future, we should probably write some benchmarks that allow us to compare db config changes quantitatively.